runtime/tests/runtime_integration_tests/test_warm_reset.rs

```diff
@@ -133,6 +135,23 @@ fn test_mbox_busy_during_warm_reset() {
     // Wait for boot
     model.step_until(|m| m.soc_ifc().cptra_flow_status().read().ready_for_runtime());
 
+    let cmd = MailboxReqHeader {
+        chksum: caliptra_common::checksum::calc_checksum(
+            u32::from(CommandId::SELF_TEST_START),
```

Comment:
I think if you use the FIPS SHUTDOWN command, it should error in lib.rs on line 304, right before we set mailbox_flow_done. So then if you warm reset after a shutdown, we should successfully hit the attestation disable logic.

Reply:
Discussed offline that this doesn't work